pennylane/__init__.py

```diff
@@ -133,6 +136,63 @@ def device(name, *args, **kwargs):
     )
 
 
+def load(quantum_circuit_object, name: str):
+    """load(name, quantum_circuit_object)
+    Load a plugin :func:`~.load` which can then be used to convert objects of
+    quantum circuits from other frameworks.
+
+    Args:
+        quantum_circuit_object: the quantum circuit that will be converted
+            to a PennyLane template
+        name (str): the name of the plugin to convert from
```

Comment:
Instead of 'name', would something like 'fmt' make more sense? From what I can see, the 'name' isn't the name of the plugin to convert from, but the format you are converting from. For example, `fmt='qasm'`, `fmt='qasm_file`, `fmt='qiskit'`.

Reply:
Let's go with `format`, no need to save space 😆

Reply:
The reason why I picked `name` is that `format` is reserved as a keyword in Python and it might raise warnings in for example IDEs. That being said it can be picked nonetheless